plugins/cicd-statistics/api-report.md

```diff
@@ -0,0 +1,182 @@
+## API Report File for "@backstage/plugin-cicd-statistics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { ApiRef } from '@backstage/core-plugin-api';
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { Entity } from '@backstage/catalog-model';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// Warning: (ae-missing-release-tag) "AbortError" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export class AbortError extends Error {}
+
+// Warning: (ae-missing-release-tag) "branchTypes" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const branchTypes: Array<FilterBranchType>;
+
+// Warning: (ae-missing-release-tag) "Build" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface Build {
+  buildType: FilterBranchType;
+  duration: number;
+  id: string;
+  // (undocumented)
+  raw?: unknown;
+  requestedAt: Date;
+  stages: Array<Stage>;
+  status: FilterStatusType;
+}
+
+// Warning: (ae-missing-release-tag) "BuildWithRaw" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export type BuildWithRaw<T = any> = Build & {
+  raw: T;
+};
+
+// Warning: (ae-missing-release-tag) "CicdConfiguration" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdConfiguration {
+  availableStatuses: ReadonlyArray<FilterStatusType>;
+  defaults: Partial<CicdDefaults>;
+  formatStageName: (parentNames: Array<string>, stageName: string) => string;
+}
+
+// Warning: (ae-missing-release-tag) "CicdDefaults" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdDefaults {
+  collapsedLimit: number;
+  // (undocumented)
+  filterStatus: Array<FilterStatusType>;
+  // (undocumented)
+  filterType: FilterBranchType<'all'>;
+  lowercaseNames: boolean;
+  normalizeTimeRange: boolean;
+  // (undocumented)
+  timeFrom: Date;
+  // (undocumented)
+  timeTo: Date;
+}
+
+// Warning: (ae-missing-release-tag) "CicdState" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdState {
+  // (undocumented)
+  builds: Array<Build>;
+}
+
+// Warning: (ae-missing-release-tag) "CicdStatisticsApi" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdStatisticsApi {
+  // (undocumented)
+  fetchBuilds(options: FetchBuildsOptions): Promise<CicdState>;
+  // (undocumented)
+  getConfiguration(): Promise<Partial<CicdConfiguration>>;
```

Comment:
Should we provide the `entity` as an option to this one as well? So that you can have different config for different CI/CD systems. Unless the config really is global, in which case it may be better to move to props on the extension instead.